src/MSRIOGroup.cpp

```diff
@@ -419,6 +451,17 @@ namespace geopm
             write_control("MSR::PKG_POWER_LIMIT:PL1_LIMIT_ENABLE", domain_type, domain_idx, 1.0);
         }
 
+        //HWP Handling
```

Comment:
Same general approach as POWER_PACKAGE_LIMIT writes implying the P1_LIMIT_ENABLE, but here we need to push the related controls since they require adjusting whenever the user adjusts the alias (which ties to MINIMUM_PERFORMANCE)